users/views.py

```diff
@@ -87,7 +87,7 @@ def users(request):
     if query == None:
         query = ''
 
-    users = User.objects.filter(Q(userprofile__name__icontains=query) | Q(userprofile__username__icontains=query))
+    users = User.objects.filter(Q(userprofile__name__icontains=query) | Q(userprofile__username__icontains=query)).select_related('userprofile').prefetch_related('userprofile__followers')
```

Comment:
could you explain this change and why we needed it?  I assume it was the performance tuning?